src/coreclr/src/pal/src/init/pal.cpp

```diff
@@ -1289,22 +1426,27 @@ static LPWSTR INIT_ConvertEXEPath(LPCSTR exe_path)
     INT return_size;
     struct stat theStats;
 
-    if (!strchr(exe_path, '/'))
+    if (strchr(exe_path, '/'))
```

Comment:
Why do we even want to look at `exe_path`? Should we rather always use the other method?

Reply:
I just wanted to preserve the old behavior for the case when an absolute path was specified. But we obviously don't have to do that.

Reply:
Less code is always better...